deepchem/utils/dft_utils/test/test_config.py

```diff
@@ -0,0 +1,12 @@
+"""
+Test for DFT Config Utilities
+"""
+
+from deepchem.utils.dft_utils.config import config
+
+
+def test_config():
+    conf = config
+    assert conf.THRESHOLD_MEMORY == 10 * 1024**3
```

Comment:
are the config variables required to be constant? Or can the user try to change them as well?

Reply:
It looks like they used it as an overall configuration. I didn't find case of modification.

Reply:
and I found some cases in which they have modified these.
`dqc/test/benchmarks.py: config.VERBOSE = 1`
and 2 more locations.

Reply:
Let's merge in for now, but we may need to come back later and clean this up